### PR TITLE
Expose --cache and --max-sql-memory flag values

### PR DESCRIPTION
**Pull request by george-angel**
Can be set via envvars

### CODE REVIEW

**Review by sbuliarca — APPROVED** [review]

File: base/statefulset.yaml [hunk 1]
```diff
@@ -89,12 +89,26 @@ spec:
...
+              exec /cockroach/cockroach start
+              --logtostderr=WARNING
+              --certs-dir /cockroach/cockroach-certs
+              --listen-addr=:26357
+              --sql-addr=:26257
+              --advertise-addr $(hostname -f)
+              --http-addr 0.0.0.0
+              --join cockroachdb-0.cockroachdb:26257,cockroachdb-0.cockroachdb:26357,cockroachdb-1.cockroachdb:26257,cockroachdb-1.cockroachdb:26357,cockroachdb-2.cockroachdb:26257,cockroachdb-2.cockroachdb:26357
+              --cache $(CACHE)
+              --max-sql-memory $(MAX_SQL_MEMORY)
```
Comment (matthewhughes-uw) [inline base/statefulset.yaml:102]: Should these env vars be documented somewhere (in the README?)?

Comment (george-angel) [reply base/statefulset.yaml:102]: Yes? I don't know how much context you want to add though, like "these values are good with X requests/limits" and where it starts to step on the toes of the info in "documentation" repo.

Comment (matthewhughes-uw) [reply base/statefulset.yaml:102]: > Yes? I don't know how much context you want to add though, like "these values are good with X requests/limits" and where it starts to step on the toes of the info in "documentation" repo.

I was just thinking to document their existence: e.g."The value of `--max-sql-memory` can be set by the `MAX_SQL_MEMORY` environment variable"

Comment (george-angel) [reply base/statefulset.yaml:102]: https://github.com/utilitywarehouse/cockroachdb-manifests/pull/81

**Pull request merged**